packages/shared/src/lib/core/transactions/actions/fetchAndPersistTransactionsForAccounts.ts

```diff
@@ -35,7 +36,12 @@ export async function fetchAndPersistTransactionsForNetwork(
 ): Promise<void> {
     for (const account of accounts) {
         try {
-            const blockscoutTransactions = await fetchBlockscoutTransactionsForAccount(profileId, account, network)
+            // eslint-disable-next-line @typescript-eslint/no-unused-vars
+            const [blockscoutTransactions, novesTransactions] = await Promise.all([
```

Comment:
If one promise rejects then this will not return the any values here. We need to think of a more robust way to do this potentially using Promise.allSettled.